runelite-client/src/main/java/net/runelite/client/util/ColorUtil.java

```diff
@@ -85,6 +85,17 @@ public static String toHexColor(final Color color)
 		return "#" + colorToHexCode(color);
 	}
 
+	/**
+	 * Converts a given color with alpha to its hexadecimal equivalent.
+	 *
+	 * @param color Color to get hexadecimal string from.
+	 * @return      Hexadecimal string representing the given color, in the form "#aarrggbb".
+	 */
+	public static String toAlphaHexColor(final Color color)
```

Comment:
this method name is easily confused with `colorToAlphaHexCode` and probably should just not exist at all.

Reply:
I suppose I can remove it (the naming was done for consistency with existing methods, so renaming would also add confusion). It's only used in two places in ConfigPanel.java, although those uses are within ternaries that are approaching questionable line lengths. Is there a preferred way to add line breaks to ternaries to make them more readable?

[ConfigPanel.java:423](https://github.com/runelite/runelite/pull/12354/files#diff-078b2bca651a2befbe603356afbeda841673e0706eb05b62f99e57156c826437R423) for example
```diff
- String colorHex = (alphaHidden ? ColorUtil.toHexColor(existing) : ColorUtil.toAlphaHexColor(existing)).toUpperCase();
+ String colorHex = (alphaHidden ? ColorUtil.toHexColor(existing) : "#" + ColorUtil.colorToAlphaHexCode(existing)).toUpperCase();
```